docs/usage/config-overview.md

```diff
@@ -1,12 +1,18 @@
 # Renovate configuration overview
```

Comment:
The pull request has a typo in the title. The correct spelling is `precedence`. 

But we should use simple words and phrases. How about something like this?

`docs(filename): explain order of operations for final config`

Reply:
Agree. Done.